config/initializers/sidekiq.rb

```diff
@@ -40,6 +40,11 @@
         # history is captured every 30 seconds by default
         config.retain_history(30)
       end
+
+      if Flipper.enabled? :appeals_api_sidekiq_job_duration_logging
```

Comment:
We can't rely on flipper settings in initialization files (and within class constants, etc), unfortunately. Since the initialization results  are cached during server startup, there'd be a possibility where some pods that boot before we change the flipper will behave as though it's false until they die, and others that start after will behave differently.

This should be a Settings-level flag so that specific deploys of all servers will enable/disable the flag.

Reply:
This comment actually becomes moot if we just have each job opt into duration logging :)